src/openforms/tests/e2e/test_input_validation.py

```diff
@@ -292,3 +317,209 @@ def test_max_date_fixed_value(self):
             api_value="2025-01-01",
             expected_ui_error="De opgegeven datum ligt te ver in de toekomst.",
         )
+
+
+class SingleCheckboxTests(ValidationsTestCase):
+
+    async def apply_ui_input(self, page: Page, label: str, ui_input: str | int | float):
+        await expect(
+            page.get_by_role("checkbox", name="Required checkbox")
```

Comment:
```suggestion
            page.get_by_role("checkbox", name=label)
```
this avoids hardcoding it to the label of a single test